src/plugins/WebhookManager/index.tsx

```diff
@@ -0,0 +1,162 @@
+                                    },
+                                    description: `
+                                Webhook ID: ${response.id}
+                                Webhook Token: ${response.token}
+                                Webhook Type: ${response.type}
+                                Channel ID: ${response.channel_id}
+                                Server ID: ${response.guild_id}`
+                                }]
+                        });
+                    });
+            }
+        },
+        {
+            name: "webhook send",
+            description: "Send a message through a webhook.",
+            inputType: ApplicationCommandInputType.BUILT_IN,
+            options: [
+                {
+                    name: "url",
+                    description: "The URL of the webhook",
+                    type: ApplicationCommandOptionType.STRING,
+                    required: true
+                },
+                {
+                    name: "message",
+                    description: "The message you want to send",
+                    type: ApplicationCommandOptionType.STRING,
+                    required: true
+                },
+                {
+                    name: "username",
+                    description: "Send with a custom username",
+                    type: ApplicationCommandOptionType.STRING,
+                    required: false
+                },
+                {
+                    name: "tts",
+                    description: "Send with TTS",
+                    type: ApplicationCommandOptionType.BOOLEAN,
+                    required: false
+                },
+                {
+                    name: "pfp",
+                    description: "Send with a custom profile picture. You must input a valid image URL.",
+                    type: ApplicationCommandOptionType.STRING,
+                    required: false
+                },
+                {
+                    name: "raw",
+                    description: "Send message as raw JSON",
+                    type: ApplicationCommandOptionType.BOOLEAN,
+                    required: false
+                }
+            ],
+            execute: async (option, ctx) => {
+
+                const webhookUrl = findOption(option, "url");
+                const webhookMessage = findOption(option, "message");
+                let webhookProfilePic = findOption(option, "pfp");
+                let webhookUsername = findOption(option, "username");
+                if (findOption(option, "raw")) {
+                    Native.executeWebhook("" + webhookUrl, {
+                        webhookMessage
+                    });
+                }
+                else {
+                    if (webhookUsername === "")
+                        webhookUsername = undefined;
+
+                    if (webhookProfilePic === "")
+                        webhookProfilePic = undefined;
```

Comment:
the default value if no fallback is provided (third argument of findOption) is already undefined, so this shouldn't be necessary
```suggestion
                const webhookProfilePic = findOption(option, "pfp");
                const webhookUsername = findOption(option, "username");
                if (findOption(option, "raw")) {
                    Native.executeWebhook("" + webhookUrl, {
                        webhookMessage
                    });
                }
                else {
```